src/js/utils/index.d.ts

```diff
@@ -185,6 +191,7 @@ export type BorderType =
       side?: BoxSideType;
       size?: BoxSizeType;
       style?: BoxStyleType;
+      image?: string;
```

Comment:
Is there a corresponding change to propTypes?

Reply:
yes changing the structure a little so I will update :)